yt_dlp/extractor/tiktok.py

```diff
@@ -323,38 +336,32 @@ def _real_extract(self, url):
         raise ExtractorError('Video not available', video_id=video_id)
 
 
-class TikTokUserIE(InfoExtractor):
+class TikTokUserIE(TikTokBaseIE):
     IE_NAME = 'tiktok:user'
-    _VALID_URL = r'https?://(?:www\.)?tiktok\.com/@(?P<id>[\w\._]+)/?(?:$|[#?])'
+    _VALID_URL = r'https?://(?:www\.)?tiktok\.com/@(?P<id>[\w\.-]+)/?(?:$|[#?])'
```

Comment:
are u sure about this change?
I think when i added this, i found some accounts using `_` in name

Reply:
`\w` catches `_` as well